bridges/otelzerolog/hook.go

```diff
@@ -0,0 +1,153 @@
+package otelzerolog
```

Comment:
This is missing a copyright header.

There seem to be other lint issues with this PR. Can you run the `make precommit` hook and update?

Reply:
Okay will do just that